insights/v1alpha1/types_insights.go

```diff
@@ -235,19 +299,23 @@ type ObjectReference struct {
 	// This value should consist of only lowercase alphanumeric characters, hyphens and periods.
 	// Example: "", "apps", "build.openshift.io", etc.
 	// +kubebuilder:validation:Pattern:="^$|^[a-z0-9]([-a-z0-9]*[a-z0-9])?(.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$"
+	// +kubebuilder:validation:MaxLength=512
```

Comment:
Why 512? should this be CEL that validates kubernetes group patterns? For example: https://github.com/kubernetes/kubernetes/blob/88d2355c412f7fa5b220ff1cf63a97e81d227da3/staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go#L353-L359

Reply:
I have changed that to follow the dns1123 format